holoviews/streams.py

```diff
@@ -267,6 +267,14 @@ def __str__(self):
         return repr(self)
 
 
+class Next(Stream):
+    """
+    Example of an empty stream that can be used to trigger generators or
+    callbacks that take no arguments.
+    """
+    pass
```

Comment:
Not strictly necessary as you could use ``Stream``. Another possible name could be ``Trigger``.